### PR TITLE
Get() is no longer part of flag.Value

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: acceptance/localcluster/localcluster.go [hunk 1]
```diff
@@ -38,11 +38,11 @@ const (
...
-var cockroachImage = flag.String("i", builderImage, "the docker image to run")
-var cockroachBinary = flag.String("b", defaultBinary(), "the binary to run (if image == "+builderImage+")")
-var cockroachEntry = flag.String("e", "", "the entry point for the image")
-var waitOnStop = flag.Bool("w", false, "wait for the user to interrupt before tearing down the cluster")
-var logDirectory = flag.String("l", "", "the directory to store log files, relative to where the test source")
+var cockroachImage = pflag.String("i", builderImage, "the docker image to run")
+var cockroachBinary = pflag.String("b", defaultBinary(), "the binary to run (if image == "+builderImage+")")
+var cockroachEntry = pflag.String("e", "", "the entry point for the image")
+var waitOnStop = pflag.Bool("w", false, "wait for the user to interrupt before tearing down the cluster")
+var logDirectory = pflag.String("l", "", "the directory to store log files, relative to where the test source")
```
Comment (petermattis) [inline acceptance/localcluster/localcluster.go]: These flags won't be usable from the "go test" command line. That seems like a serious drawback. Ditto for the logging flags.

Comment (tamird) [inline acceptance/localcluster/localcluster.go]: Oh, because nothing calls Parse? Damnit.

**Comment by tschottdorf** [comment]
I had that in one of my previous PRs and we decided then that the status quo was the better option.

**Comment by tamird** [comment]
OK :(

Just removing some dead code then. PTAL

**Comment by tschottdorf** [comment]
There's no shame in dreaming. LGTM

**Pull request merged**